EpiAware/README.md

```diff
@@ -81,3 +89,13 @@ H-.->I
 ## Pluto scripts
 
 We use [`Pluto.jl`](https://plutojl.org/) scripts as part of our documentation and testing. The scripts are located in `docs/src/examples` and can be run using the `Pluto.jl` package. We recommend using the version of `Pluto` that is pinned in the `Project.toml` file defining the documentation environment. An entry point to running or developing this documentation is the `docs/pluto-scripts.sh` bash shell script. Run this from the root directory of this repository.
+
+## Useful Julia packages for global environment (optional and opinionated)
+In our view these packages are useful for a global environment available to other environments (e.g. add to the `@v1.10` environment), but are not required for the `EpiAware` package to work.
+
+- `Revise`: For modifying package code and using the changes without restarting Julia session.
```

Comment:
Do we want to go further and suggest anything for the startup file? (i.e `Term` and `Revise`?)

Reply:
Nice idea, added a bit on this.